esx_service/utils/auth_data.py

```diff
@@ -410,7 +422,8 @@ def create_default_tenant(self):
                                            name=auth.DEFAULT_TENANT, 
                                            description="This is a default tenant", 
                                            vms=[], 
-                                           privileges=[])
+                                           privileges=[],
+                                           tenant_uuid=auth.DEFAULT_TENANT_UUID)
```

Comment:
Even if we may (I guess not) introduce some other special tenants in the future, we can still follow the same approach, i.e. handle the special tenant_name to tenant_uuid mapping transparently inside the create_tenant() method, hiding this detail from the client code.